contributor/Testing_Request.md

```diff
@@ -0,0 +1,7 @@
+# Testing Request
+The OpenLab provides two kinds of resources: VM(Virtual Machine) and PM(Physical Machine) for testing. Currently we only utilize virtual machines to support automating jobs for SDKs integration test with OpenLab CI. On the other hand you can request PMs for your other testing.
+
+* Submit a test request to OpenLab, fill out some information like what kind of machines (VMs or PMs) and how many do you expect to use, what testing are you planning to implement or need assistance implementing and so on, for more details see [ocean request](https://github.com/theopenlab/ocean/issues/new) 
+* The governance team owner will confirm the request to check whether it is acceptable, if yes then will add it to [TODO List](https://github.com/orgs/theopenlab/projects/1#column-1860008)
+* The test request in TODO list can be assigned, if you wish to work on it please pick it up, move the item to [IN PROGRESS List](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit the item to add the development plan
+* Begin your development and test if you picked up the ocean request, and follow [Development Guide](./Development_Workflow.md)
```

Comment:
Add an item about "Share your testing result", include:
1. If you use OpenLab CI, the test result will be shown in http://status.openlabtesting.org/builds automatically
2. If you use physical machine to test, please commit the test result into comments of testing request related issue. A blog also is a good way to show more details of testing.